[BUG] O programa lipsum exibe uma linha a menos que o solicitado

:bug:  Bug report

**Descreva o bug**

O programa lipsum às vezes exibe uma linha a menos o que solicitado.
Rodando o programa lipsum a partir de um Makefile, chamando 'lipsum -r 10' o seguinte erro é gerado:

```sh
~$ make test
Testando a biblioteca Shellinclude...
...Testando ambiente
...Testando lipsum
	Exibir uma linha com lipsum -r ...passou 1
sed: -e expression #1, char 2: invalid usage of line address 0
	Exibir 10 linhas com lipsum -r 10 lipsum_tdd.sh: ERRO(2): Teste falhou. 
 Resultado esperado: 10
Resultado obtido: 9 
make: *** [Makefile:30: lipsum] Error 2
```

**Como reproduzir o bug**

O erro é provavelmente gerado quando o programa sed recebe o número de linha '0' para imprimir.
Já que o programa lipsum imprime uma linha aleatória de um arquivo de configuração, se esta linha for zero, o sed apresentará o erro.

Passos para reproduzir o bug:
1. O mesmo erro aparece quando tentar imprimir a linha número 0 de um arquivo com sed:

```sh
~$ sed -n '0p' .lipsum/dummy.txt 
sed: -e expression #1, char 2: invalid usage of line address 0
```

**Comportamento esperado**

Alterar o intervalo de exibição de linhas do arquivo. De 1 até o número de linhas do arquivo.

**Desktop (Complete a descrição a seguir):**

 - SO: Ubuntu
 - Versão 19.10

--- a/TDD/lipsum_tdd.sh
+++ b/TDD/lipsum_tdd.sh
@@ -19,3 +19,16 @@ source $(dirname $0)/tdd_lib.sh
 # Testar se lipsum exibe a quantidade de linhas pedidas corretamente
 error "$(lipsum -r | wc -l)" "1" "1" "Exibir uma linha com lipsum -r"
 error "$(lipsum -r 10 | wc -l)" "10" "2" "Exibir 10 linhas com lipsum -r 10"
+
+# Testar condição limite
+total="1000"
+for i in $(seq $total)
+do
+	echo -ne "Rodando lipsum repetidamente ($i/$total)\r\r"
+	LINHAS=$(lipsum -r | wc -l)
+	if [ "$LINHAS" -ne "1" ]
+	then
+		echo " "
+		error "$LINHAS" "1" "3" "Exibir uma linha com lipsum -r"
+	fi
+done

--- a/TDD/lipsum_tdd.sh
+++ b/TDD/lipsum_tdd.sh
@@ -24,7 +24,7 @@ error "$(lipsum -r 10 | wc -l)" "10" "2" "Exibir 10 linhas com lipsum -r 10"
 total="1000"
 for i in $(seq $total)
 do
-	echo -ne "Rodando lipsum repetidamente ($i/$total)\r\r"
+	echo -ne "\tRodando lipsum repetidamente ($i/$total)\r\r"
 	LINHAS=$(lipsum -r | wc -l)
 	if [ "$LINHAS" -ne "1" ]
 	then
@@ -32,3 +32,5 @@ do
 		error "$LINHAS" "1" "3" "Exibir uma linha com lipsum -r"
 	fi
 done
+
+error "$LINHAS" "1" "3" "Rodando lipsum repetidamente ($total/$total)"

--- a/lipsum
+++ b/lipsum
@@ -102,7 +102,7 @@ case "$1" in
 
 		for i in $(seq "$NUM_LINHAS")
 		do
-			LINHA=$(echo "$RANDOM % $TOTAL_LINHAS" | bc )
+			LINHA=$(echo "$RANDOM % $TOTAL_LINHAS + 1" | bc )
 			sed -n "${LINHA}p" "$CONFIG"
 		done
 	;;

